configuration/etl/etl_macros.d/resource_specs_time_diff.sql

```diff
@@ -1 +1 @@
-GREATEST(0, 1 + COALESCE(LEAST(r.end_date_ts, ${:PERIOD_END_TS}), ${:PERIOD_END_TS}) - GREATEST(r.start_date_ts, ${:PERIOD_START_TS}))
+GREATEST(0, 1 + COALESCE(LEAST(ra.end_date_ts, r.end_date_ts, ${:PERIOD_END_TS}), ${:PERIOD_END_TS}) - GREATEST(ra.start_date_ts, r.start_date_ts, ${:PERIOD_START_TS}))
```

Comment:
```suggestion
GREATEST(
    0,
    1 + COALESCE(
        LEAST(
            ra.end_date_ts,
            r.end_date_ts,
            ${:PERIOD_END_TS}
        ),
        ${:PERIOD_END_TS}
    ) - GREATEST(
        ra.start_date_ts,
        r.start_date_ts,
        ${:PERIOD_START_TS}
    )
)
```
To help with readability.